internal/repository/rsl_test.go

```diff
@@ -211,20 +211,7 @@ func TestRollbackLatestRSLEntry(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	blobID, err := repo.r.WriteBlob([]byte("test"))
-	if err != nil {
-		t.Fatal(err)
-	}
-	treeWithContentsID, err := treeBuilder.WriteRootTreeFromBlobIDs(map[string]gitinterface.Hash{"README.md": blobID})
-	if err != nil {
-		t.Fatal(err)
-	}
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if _, err := repo.r.Commit(treeWithContentsID, refName, "Second commit", false); err != nil {
+	if _, err := repo.r.Commit(emptyTreeHash, refName, "Second commit", false); err != nil {
```

Comment:
could you explain the change in test here, sorry?

Reply:
I realized that I did not need to actual commit real data, so I thougth this would make it easier to understand